prime-router/src/main/kotlin/Report.kt

```diff
@@ -213,8 +213,10 @@ class Report {
             }
             toElement.isCode -> {
                 Array(table.rowCount()) { row ->
-                    val fromDisplay = table.getString(row, toElement.nameAsCodeText)
-                    valueSet.toCode(fromDisplay) ?: toElement.default ?: ""
+                    // Value in the table is already a code.  Just return it; no display->code mapping needed.
+                    // BUG!! If value in the incoming table is a #text display, this breaks!
+                    // Need robust generic way to have many ways to display a single valueset.
+                    table.getString(row, toElement.nameAsCode) ?: "BUG - incoming schema used #text?"
```

Comment:
Example of the bug:  if some incoming PDI schema field uses "#text", that is, it sends in a 'display' value, and the outgoing AZ schema does not use "#text", that is, it expects a 'code' value, then this code will break.   By sheer luck, current incoming schema has no "#text", so I'm momentarily punting on fixing this, esp as I think a more general solution is needed.

Reply:
Could you change this to a TODO which can be found by IntelliJ?

Reply:
will do